pom.xml

```diff
@@ -32,6 +32,7 @@
   </parent>
 
   <properties>
+    <project.version>KeepTime-1.2.0-SNAPSHOT</project.version>
```

Comment:
this is not a valid version.
https://maven.apache.org/pom.html#maven-coordinates

Reply:
I removed "KeepTime" from the version because it was doubled. But I am not sure if that was the thing you meant

Reply:
yes, that was the issue.